src/rules/core/value-in-options-rule.js

```diff
@@ -40,6 +50,7 @@ module.exports = class ValueInOptionsRule extends Rule {
       && typeof fieldObj !== 'undefined'
     ) {
       let isInOptions = true;
+      let cannotBeArray = false;
```

Comment:
Perhaps the naming of this var should be slightly clearer?

```suggestion
      let isArrayButCannotBeArray = false;
```